scripts/sous/setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Should this end in `sh` instead of `bash` specifically to allow for other shells?